### DISCUSSION TITLE
Why is the power limit (PL1 + PL2) combined on Intel chips?

### DISCUSSION THREAD

**Discussion by 47chromo** [seerge/g-helper#1064] [General]
Just wondering why the power limit is combined when it is actually separate in the original Armoury Crate software for slow and fast loads. Also was wondering how even setting a power limit of around 35W for both PLs still results in the CPU going up to 50W and maximum clock speeds.

**Comment by seerge** [comment]
@47chromo they are combined for simplicity :) as majority have no idea what PL1 and PL2 is (and constantly ask same about armoury crate manual mode settings)

As for power going above - it could be that some devices have some minimum hardcoded limit, and going below that is not really possible. What device do you have ?

**Reply by 47chromo** [reply]
F15 2023 so I have an i7 12700H. I'm assuming that the CPU somehow ignores the 35W limit to reach the Asus 95 degree temp limit?

**Reply by seerge** [reply]
In some exotic cases, device may need a special "manual" mode toggled to be able to limit power.

Try to quit g-helper, add to ``%appdata%\ghelper\config.json`` following line, save
```
"manual_mode" : 1,
```

and run g-helper again, and try to apply limits

**Reply by 47chromo** [reply]
Thanks for the reply I'll report back once I have the time to benchmark some games

**Reply by seerge** [reply]
Easiest way to benchmark is just to run cinebench23 and monitor with HWinfo

**Reply by 47chromo** [reply]
Ok so doing the manual mode edit completely fixed my issue thank you brodie

**Reply by seerge** [reply]
What’s your full model name? I can hardcode in the list that requires manual mode - so you won’t need config edits

**Reply by 47chromo** [reply]
F15 FX507ZV4

**Reply by 47chromo** [reply]
hey I think in the latest update the settings for the Windows power mode change for some reason whenever you switch between saved profiles

**Reply by seerge** [reply]
Power mode was always changing with profiles, now you can control and see that via UI

**Reply by 47chromo** [reply]
my issue is that the power mode keeps changing on my custom profile (to best performance) every time I switch to it despite setting it at balanced

**Reply by seerge** [reply]
Set in under fans+power.

Also make sure that your custom power plan (why do you use it?) allows changing power modes. As some have it locked

**Reply by 47chromo** [reply]
hey the saving thing was fixed in the new update but I've been having randomly high fan usage and temps seemingly when doing nothing? On the exact same mode as before and the latest 0.112 update. The app also for some reason seems jankier when installing because it opens and blanks just to reopen again

**Reply by seerge** [reply]
I haven’t changed anything related to that :) your fans are controlled by bios

**Reply by 47chromo** [reply]
gotcha probably on my end then

**Comment by AudreyAP** [comment]
Would it be possible to add an option to either config.json or the power menu to separate them? Default together is a good idea but would be nice to be able to adjust.

**Reply by IceStormNG** [reply]
Tbh. The PL1/PL2 setting from GHelper is quite limited as it uses the ASUS Driver Endpoints (which go through the firmware). Using throttlestop would be way easier and more likely to work as it writes directly to MMIO and/or MSR and ignores whatever the BIOS is doing there.

**Reply by AudreyAP** [reply]
I didn't see the ability to adjust anything in throttle stop but I'll test again. Even so, nice to use only one piece of software if it works.

**Reply by IceStormNG** [reply]
The settings are in the TPL window. 
<img width="308" alt="image" src="https://github.com/seerge/g-helper/assets/12786283/4b549c06-d5eb-4a59-8b69-0b8ecc7aba23">
